Content.Server/GameObjects/Components/ListeningComponent.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Content.Server.GameObjects.Components.Interactable;
+using Content.Server.GameObjects.EntitySystems;
+using Content.Server.Interfaces;
+using Robust.Shared.GameObjects;
+using Robust.Shared.GameObjects.Components;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+
+namespace Content.Server.GameObjects.Components
+{
+    [RegisterComponent]
+    public class ListeningComponent : Component
+    {
+#pragma warning disable 649
+        [Dependency] private readonly IEntitySystemManager _entitySystemManager = default!;
+#pragma warning restore 649
+
+        public override string Name => "Listening";
+
+        private ListeningSystem _listeningSystem = default!;
```

Comment:
Remove _listeningSystem reference here and do what was done for RadioComponent to replace the subscribe unsubscribe.